### PR TITLE
OS9 Ansible Collections

### PR DESCRIPTION
**Pull request by komalupatil**
##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by skg-net** [comment]
@gundalow  can you please provide with a merge access to the os9 collection.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
+                     Apache License
```
Comment (gundalow) [inline LICENSE:1]: The original code was under GPLv3+, please check this with your legal team

File: plugins/modules/os9_command.py [hunk 2]
```diff
@@ -25,32 +26,27 @@
...
   wait_for:
     description:
       - List of conditions to evaluate against the output of the
         command. The task will wait for each condition to be true
         before moving forward. If the conditional is not true
-        within the configured number of retries, the task fails.
+        within the configured number of I(retries), the task fails.
         See examples.
     type: list
+    version_added: "2.2"
```
Comment (gundalow) [inline plugins/modules/os9_command.py:49]: Please remove all `version_added` lines from modules and modules options.

**Comment by gundalow** [comment]
You should have commit now

**Pull request merged**